tests/ui_tests/app/page/cluster/locators.py

```diff
@@ -113,3 +116,45 @@ class Row:
         class RelationsRow:
             name = Locator(By.XPATH, "./div/span", "Related item name")
             delete_btn = Locator(By.XPATH, ".//button", "Delete item button")
+
+
+class ClusterConfigLocators:
+    """Cluster config page elements locators"""
+
+    description_input = Locator(
+        By.XPATH, "//input[@data-placeholder='Description configuration']", "Description input"
+    )
+    search_input = Locator(By.XPATH, "//input[@data-placeholder='Search field']", "Search input")
+    search_input_clear_btn = Locator(
+        By.XPATH, "//app-search//button[@aria-label='Clear']", "Clear search input button"
+    )
+    advanced_chbx = Locator(
+        By.XPATH, "//mat-checkbox[.//*[contains(text(), 'Advanced')]]", "Checkbox Advanced"
+    )
+    save_btn = Locator(
+        By.XPATH, "//button[contains(@class, 'form_config_button_save')]", "Save button"
+    )
+    history_btn = Locator(By.XPATH, "//button[.//*[contains(text(), 'history')]]", "history button")
+
+    row = TemplateLocator(By.XPATH, "//app-field", "Configuration row")
+    group = TemplateLocator(By.XPATH, "//mat-expansion-panel", "Configuration group")
+    row_template = TemplateLocator(
+        By.XPATH, "//div[@adcm_test='{}']", "Configuration row with name {}"
+    )
```

Comment:
What does "row" mean?
And don't we have these locators somewhere in common/configuration?

Reply:
changed